### PR TITLE
[[ Datagrid ]] Add datagrid docs to docs build

### PR DESCRIPTION
**Pull request by livecodeali**
Requires https://github.com/livecode/livecode-ide/pull/1218

### CODE REVIEW

**Comment by peter-b** [comment]
@livecode-vulcan review ok 76860e2

**Comment by livecode-vulcan** [comment]
💙 review by @peter-b ok 76860e2795dca553c370964927b40b9b763c816b

**Comment by livecode-vulcan** [comment]
😎 test success 76860e2795dca553c370964927b40b9b763c816b
- try-community-linux-x86: [success](https://vulcan.livecode.com/builders/try-community-linux-x86/builds/2599)
- try-community-mac: [success](https://vulcan.livecode.com/builders/try-community-mac/builds/2609)
- try-community-linux-x86_64: [success](https://vulcan.livecode.com/builders/try-community-linux-x86_64/builds/2594)
- try-community-emscripten: [success](https://vulcan.livecode.com/builders/try-community-emscripten/builds/2610)
- try-community-ios: [success](https://vulcan.livecode.com/builders/try-community-ios/builds/2616)
- try-community-win-x86: [success](https://vulcan.livecode.com/builders/try-community-win-x86/builds/2622)
- try-community-android-armv6: [success](https://vulcan.livecode.com/builders/try-community-android-armv6/builds/2634)

File: builder/docs_builder.livecodescript [hunk 1]
```diff
@@ -2194,6 +2194,20 @@ command docsBuilderGenerateDistributedAPI pEdition
...
+
+   if tDataGridA is empty then
+      builderLog "error", "Couldn't parse datagrid dictionary data for" && pEdition
+      exit docsBuilderGenerateDistributedAPI
+   end if
+   
+   put "dictionary" into tDataGridA["type"]
+   
+   addToList tLCBDictionaryA, tLibrariesA
+
```
Comment (livecodepanos) [inline builder/docs_builder.livecodescript:2210]: I think this should be:
`addToList tDataGridA, tLibrariesA`

Comment (peter-b) [inline builder/docs_builder.livecodescript:2210]: That would explain the multiple-copies-of-LCB-docs bug! Sorry for missing this in review...

Comment (peter-b) [inline builder/docs_builder.livecodescript:2210]: Fix is in #4132.

**Pull request merged**